generators/generator-base.js

```diff
@@ -46,8 +46,8 @@ Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enab
             file: fullPath,
             needle: 'jhipster-needle-add-element-to-menu',
             splicable: [
-                '<li ui-sref-active="active" >\n' +
-                '                    <a ui-sref="' + routerName + '" ng-click="vm.collapseNavbar()">\n' +
+                '<li class="nav-item" ui-sref-active="active" >\n' +
```

Comment:
this shouldnt be changed at the moment ill do this after entity migration. same for all changes on this file